contracts/contracts/strategies/NativeStaking/FeeAccumulator.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { Address } from "@openzeppelin/contracts/utils/Address.sol";
+
+import { Governable } from "../../governance/Governable.sol";
+
+/**
+ * @title Fee Accumulator for Native Staking SSV Strategy
+ * @notice Receives execution rewards which includes tx fees and
+ * MEV rewards like tx priority and tx ordering.
+ * It does NOT include swept ETH from beacon chain consensus rewards or full validator withdrawals.
+ * @author Origin Protocol Inc
+ */
+contract FeeAccumulator is Governable {
+    /// @notice The address of the Native Staking Strategy
+    address public immutable STRATEGY;
+
+    /**
+     * @param _strategy Address of the Native Staking Strategy
+     */
+    constructor(address _strategy) {
+        STRATEGY = _strategy;
+    }
+
+    /**
+     * @notice sends all ETH in this FeeAccumulator contract to the Native Staking Strategy.
+     * @return eth The amount of execution rewards that were sent to the Native Staking Strategy
+     */
+    function collect() external returns (uint256 eth) {
+        require(msg.sender == STRATEGY, "Caller is not the Strategy");
+
+        eth = address(this).balance;
+        if (eth > 0) {
```

Comment:
You have a couple of superior to zero checks, it's more gas-efficient to use a not zero check.

Reply:
I've tested this with the [hardhat-gas-reporter](https://www.npmjs.com/package/hardhat-gas-reporter) and Remix and didn't see a noticeable difference in gas usage.